### PR TITLE
Add Ability to Follow Other Users

### PR DESCRIPTION
**Pull request by daaray**
This PR encompasses the basic Follow/Unfollow functionality; account_profile_related needs layout/styling.

### CODE REVIEW

File: raspberryio/userprofile/views.py [hunk 1]
```diff
@@ -1 +1,36 @@
-# Create your views here.
+from django.shortcuts import render, get_object_or_404, redirect
+
+from relationships.models import RelationshipStatus
+from relationships.views import get_relationship_status_or_404
+
+from raspberryio.userprofile.models import Profile
+
+
```
Comment (calebsmith) [inline raspberryio/userprofile/views.py:7]: This is the only part I'm confused about. The url pattern requires a relation string, but the view allows it to be optional and has a fallback 'if not relation:'. Can we just require `relation` or am I missing something?

Also, does extra_context ever get passed in?

Maybe there is something in the template filter from django-relationships that requires one or both of these on the view?

Comment (daaray) [inline raspberryio/userprofile/views.py:7]: @calebsmith: I cleaned up view based on feedback.

File: raspberryio/static/js/update_relationship.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+$(document).ready(function()
```
Comment (calebsmith) [inline raspberryio/static/js/update_relationship.js:1]: I think we could clean up the $.post() statements a little using chaining. For example

```
$.post($item).attr('href'), function() {
    var $item = $(this);
    $item.attr('class', 'follow')
        .text('Follow')
        .attr ...
if that makes sense (and actually works)
});
```

**Comment by calebsmith** [comment]
I think the merge is failing because develop has seen some changes to templates/accounts/account_profile.html

**Pull request merged**